packages/react-query-sdk/src/notifications/services/V1Service.ts

```
@@ -1089,6 +1089,460 @@ export interface V1Service {
     };
   };
+  getContactPoints: {
+    /**/
+    cancelQueries<TInfinite extends boolean = false>(
+      filters?:
+        | QueryFiltersByParameters<GetContactPointsSchema, GetContactPointsData, TInfinite, GetContactPointsParameters, GetContactPointsError>
+        | QueryFiltersByQueryKey<GetContactPointsSchema, GetContactPointsData, TInfinite, GetContactPointsParameters, GetContactPointsError>,
+      options?: CancelOptions
+    ): Promise<void>;
+    /**/
+    getQueryKey(parameters: DeepReadonly<GetContactPointsParameters> | void): ServiceOperationQueryKey<GetContactPointsSchema, GetContactPointsParameters>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)